src/font.c

```diff
@@ -705,7 +705,8 @@ PangoFontDescription*
 gdip_get_pango_font_description (GpFont *font)
 {
 	if (!font->pango) {
-		font->pango = pango_font_description_from_string ((char*)font->face);
+		font->pango = pango_font_description_new ();
+		pango_font_description_set_family (font->pango, (char *)font->face);
```

Comment:
are you sure this does everything that `pango_font_description_from_string ()` did before? Not sure I fully grok the reason for this change.

Reply:
It does not, which is kind of the point. `pango_font_description_from_string` does some advanced parsing of the string to support multiple families or size specification. `font->face` is copy of the name in GpFontFamily, which is what the caller really asked for.

Reply:
ok :)